admin_guide/diagnostics_tool.adoc

```diff
@@ -157,36 +161,133 @@ A failure from this check indicates that a significant amount of space in etcd
+Elasticsearch.
+
+A user-defined timeout may be set by passing the
+`openshift_check_logging_index_timeout_seconds` variable. For example, setting
+`openshift_check_logging_index_timeout_seconds=30` will cause the check to fail
+if a newly-created Kibana log is not able to be queried via Elasticsearch after
+30 seconds.
+
+|`ovs_version`
+|This check ensures that a host has the correct version of Open vSwitch installed
+for the currently deployed version of {product-title}.
+|===
+
+[NOTE]
+====
 A similar set of checks meant to run as part of the installation process can be
 found in
 xref:../install_config/install/advanced_install.adoc#configuring-cluster-pre-install-checks[Configuring Cluster Pre-install Checks]. Another set of checks for checking certificate
 expiration can be found in
 xref:../install_config/redeploying_certificates.adoc#install-config-redeploying-certificates[Redeploying Certificates].
+====
+
+[[admin-guide-health-checks-via-ansible-playbook]]
+=== Running Health Checks via ansible-playbook
+
+To run the *openshift-ansible* health checks using the `ansible-playbook`
+command, specify your cluster's inventory file and run the *_health.yml_*
+playbook:
+
+----
+# ansible-playbook -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-checks/health.yml
+----
+
+To disable specific checks, include the variable `openshift_disable_check` with
+a comma-delimited list of check names in your inventory file before running the
+playbook. For example:
+
+----
+openshift_disable_check=ovs_version,etcd_volume
+----
+
+To set variables that accept user-define values, include the `-e` flag with any
+desired variables in `key=value key=value` format. For example:
+
+----
+# ansible-playbook -i <inventory_file> \
+    /usr/share/ansible/openshift-ansible/playbooks/common/openshift-checks/health.yml \
+    -e "openshift_check_logging_index_timeout_seconds=30 etcd_max_image_data_size_bytes=40000000000"
+----
+
+[[admin-guide-health-checks-via-docker-cli]]
+=== Running Health Checks via Docker CLI
+
+It is possible a playbook may require dependencies that are not installed
+locally on the host running the `ansible-playbook` command. You can avoid this
+issue by running the *openshift-ansible* health checks directly from the
+ifdef::openshift-enterprise[]
+*ose-ansible*
+endif::[]
+ifdef::openshift-origin[]
+*origin-ansible*
+endif::[]
+image via the Docker CLI.
+
+To run the *openshift-ansible* diagnostic checks directly via the `docker`
+command, specify your cluster's inventory file and run the *_health.yml_*
+playbook:
+
+----
+# docker run -u `id -u` \
+    -v $HOME/.ssh/id_rsa:/opt/app-root/src/.ssh/id_rsa:Z,ro \
+    -v /etc/ansible/hosts:/tmp/inventory:ro \ <1>
+    -e INVENTORY_FILE=/tmp/inventory \
+    -e PLAYBOOK_FILE=playbooks/common/openshift-checks/health.yml \ <2>
+    -e OPTS="-v -e openshift_check_logging_index_timeout_seconds=30 etcd_max_image_data_size_bytes=40000000000" \ <3>
```

Comment:
BTW are the callouts here going to interfere with copy/paste in the rendered version? Because they're really going to want to copy/paste this.